bundles/org.openhab.binding.mqtt.espmilighthub/src/main/java/org/openhab/binding/mqtt/espmilighthub/internal/ConfigOptions.java

```diff
@@ -21,9 +23,10 @@
  */
 @NonNullByDefault
 public class ConfigOptions {
+    public BigDecimal duvThreshold = BigDecimal.ONE;
```

Comment:
Isn't it strange than it is set to 1 there while it is set to 0.003 in the XML file ?

Reply:
@ccutrer ?

Reply:
oops, I missed your comment here. thanks for the ping. and yes, that is odd. should match the XML.